app/src/main/res/layout/activity_wait.xml

```diff
@@ -24,7 +24,10 @@
         android:layout_height="wrap_content"
         android:layout_below="@+id/avi"
         android:layout_centerHorizontal="true"
+        android:textAlignment="center"
         android:layout_marginTop="@dimen/waiting_for_slave_text_offset"
+        android:paddingEnd="@dimen/ten_padding"
```

Comment:
dimens.xml is meant as a way to easily change distances between certain elements. If you use ten_padding, you cannot change the distance anymore because, well, then the value is obviously not ten anymore. Additionally, it's possible that some other button that has nothing to do with this one looks good by a padding of 10 as well accidentally. However, if you want to change the padding of this button, then suddenly the padding of that other button changes as well, and because that button has nothing to do with it, you do not want that.
I think you can better use something like @dimen/text_screen_width_padding or @dimen/wait_activity_text_padding